vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -916,6 +851,67 @@ def prov_set_form_vars(req = nil)
     end
   end
 
+  def workflow_type_from_vars(req)
```

Comment:
Looking at the code, `workflow_instance_from_vars` or `workflow_object_from_vars` seems more appropriate.  With the name as it it, I expect the return value to be a class or a class name.